lib/Simplifier/DifficultyScore.cpp

```diff
@@ -54,7 +54,7 @@ long eval(const ASTNode& b)
     const auto cbv = b[0].GetBVConst(); // cleanup?
     bool last = CONSTANTBV::BitVector_bit_test(cbv,0);
     int changes = 0;
-    for (int i =1; i < b.GetValueWidth();i++)
+    for (unsigned i =1; i < b.GetValueWidth();i++)
```

Comment:
This should be `unsigned int` not "bare" `unsigned`.

Reply:
Ok! But why? You using "bare" `unsigned` literally in the suggestion above.

Reply:
No, I didn't "add" it ... the code is currently wrong and shouldn't use bare `unsigned` and should use `unsigned int`. When you're touching code, we can make it better; if you're not touching some bit of code, just leave it alone (even if it is wrong).

Reply:
"wrong" in this case is "can be improved", but minimal PRs are a lot easier to make and are more sensible.

Reply:
Got it, will fix!